src/docs/getting-started/installation.md

```diff
@@ -4,66 +4,40 @@ description: "How to install probe-rs."
 order: 20
 ---
 
+# Installing a precompiled binary (`cargo binstall`)
 
-# Prerequisites
+## Prerequisites
 
-To build probe-rs, you will need a working Rust toolchain. The easiest way is to follow the [rustup installation instructions](https://rustup.rs).
+First, set up `cargo-binstall`. You can find setup instructions [`here`](https://crates.io/crates/cargo-binstall).
 
-Additionally, probe-rs depends on [libusb](https://libusb.info/) and optionally on [libftdi](https://www.intra2net.com/en/developer/libftdi/) libraries, which need to be installed to build probe-rs. Instructions for your OS are provided below.
+## Installation
 
-## Debian-based Linux
+`cargo binstall probe-rs`
 
-On Debian and derived distros (e.g. Ubuntu), the following packages need to be installed:
+# Installing from source (`cargo install`)
 
-```bash
-sudo apt install -y pkg-config libusb-1.0-0-dev libftdi1-dev libudev-dev libssl-dev
-```
+## Prerequisites
 
-If the libusb v0.1 dev package (`libusb-dev`) is installed when dependant crates are built, you may get link failures at the end of the build.
+To build probe-rs from source, you will need a working Rust toolchain. The easiest way is to follow the [rustup installation instructions](https://rustup.rs).
 
-On Ubuntu you can fix this by removing it with
+### Debian-based Linux
 
-```bash
-sudo apt purge libusb-dev
-```
-
-## RPM-based Linux (Fedora, CentOS)
-
-```bash
-dnf install libusbx-devel libftdi-devel libudev-devel openssl-devel
-```
-
-## Windows
-
-On Windows you can use [vcpkg](https://github.com/microsoft/vcpkg/#quick-start-windows) to install the prerequisites:
+On Debian and derived distros (e.g. Ubuntu), the following packages need to be installed:
 
 ```bash
-# dynamic linking 64-bit
-vcpkg install libftdi1:x64-windows libusb:x64-windows
-set VCPKGRS_DYNAMIC=1
-
-# static linking 64-bit
-vcpkg install libftdi1:x64-windows-static-md libusb:x64-windows-static-md
+sudo apt install -y pkg-config libudev-dev
 ```
 
-## macOS
```

Comment:
Maybe we should list macOS as nothing required still? Same for Windows, just to make sure folks don't think the section is missing.